tests/vmi_multus_test.go

```diff
@@ -387,6 +373,38 @@ var _ = Describe("Multus Networking", func() {
 		})
 	})
 
+	Context("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance with ovs-cni plugin interface and custom MAC address.", func() {
+		interfaces := []v1.Interface{{Name: "default", InterfaceBindingMethod: v1.InterfaceBindingMethod{Bridge: &v1.InterfaceBridge{}}},
+			{Name: "ovs", InterfaceBindingMethod: v1.InterfaceBindingMethod{Bridge: &v1.InterfaceBridge{}}}}
+		networks := []v1.Network{{Name: "default", NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}}},
+			{Name: "ovs", NetworkSource: v1.NetworkSource{Multus: &v1.CniNetwork{NetworkName: "ovs-net-vlan100"}}}}
+		ovsIfIdx := 1
+
+		BeforeEach(func() {
```

Comment:
why we have a before each here?
we can just create two vmi's in the `It` section

Reply:
@SchSeba Originally this context was supposed to include this TC and another `It`, which covers invalid MAC, but in the end Meni implemented the second TC.
I will remove the BeforeEach and run everything inside the `It`.

Reply:
Done